src/components/MDXComponents/EmailSubscriptionForm.tsx

```diff
@@ -0,0 +1,139 @@
+import { Trans, useI18next } from "gatsby-plugin-react-i18next";
+import { useState } from "react";
+import { z } from "zod";
+
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+
+import { errorMessage } from "./email-subscription-form.module.css";
+
+type FormType = {
+  email: string;
+  loading: boolean;
+  error: null | "invalidEmail" | "networkError";
+};
+
+// Collect email address that subscribe to TiDB release and send it to the SendGrid API
+export function EmailSubscriptionForm() {
+  const { t, navigate } = useI18next();
+
+  const API_URL =
+    "https://03tryjsrrb.execute-api.us-west-2.amazonaws.com/Prod/subscribe/";
+
+  const [formData, setFormData] = useState<FormType>({
+    email: "",
+    loading: false,
+    error: null,
+  });
+  const [success, setSuccess] = useState(false);
+
+  const validateEmail = (): boolean => {
+    // Set up a schema for validating the email address
+    const schema = z.string().email();
+
+    try {
+      schema.parse(formData.email);
+      console.log("Email is valid");
+      return true;
+    } catch (e) {
+      console.error(e);
+      setFormData({ ...formData, error: "invalidEmail" });
+      return false;
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+    setFormData((prevState) => ({ ...prevState, email: value }));
+  };
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (validateEmail()) {
+      // Submitting...
+      setFormData({ ...formData, loading: true, error: null });
+
+      try {
+        await fetch(API_URL, {
+          method: "POST",
+          body: `email=${formData.email}`,
+          headers: {
+            "Content-Type": "application/x-www-form-urlencoded",
+          },
+        });
```

Comment:
We can use `axios` instead.

```suggestion
import axios from "axios";

        await axios.post(API_URL, undefined, {
          email: formData.email
        });
```